### ISSUE TITLE
Training on MSCOCO and testing on Flickr

### ISSUE THREAD

**Issue by saitarslanboun** [yufengm/Adaptive#7]
Hi,

I am training your model with MSCOCO dataset, and validating with Flickr validation data. 

After 2nd epoch, I started to have results like;

"A yellow train", "A small bus", "An aeroplane." etc.

The remaining of the sentences are missing.

Is it because of the data, or what?

Thank you,

**Comment by yufengm** [comment]
Did you validate or test on Flickr?
I haven't looked at results at early epochs. 
Typically it converges at least after epoch 20.

**Comment by saitarslanboun** [comment]
I see. Actually my model is huge, (in which I have integrated your model), and for this reason I can use at most batch size 5. In this way, I have decreased learning rate to 0.0001. 

Did you get close results to the relevant paper?

**Comment by yufengm** [comment]
Because I didn't implement the beam search, so there is still like 2point margin.

**Comment by fawazsammani** [comment]
@saitarslanboun I've implemented this paper on the Flickr30k dataset, and the trained model is provided. You can find the implementation [here](https://github.com/fawazsammani/knowing-when-to-look-adaptive-attention)
